src/components/Crypto/index.tsx

```diff
@@ -13,35 +14,50 @@ import {
   ArrowPercentage,
   PercentageChangeLast24hrs,
 } from '../Crypto/styles';
+import {deleteCrypto} from '../../store/actions';
+import {useDispatch} from 'react-redux';
 
-const Crypto: FC = ({item}: {item: CryptoTypes}) => (
-  <CryptoContainer>
-    <Logo
-      source={{
-        uri: `https://messari.io/asset-images/${item.id}/128.png`,
-      }}
-    />
-    <NameContainer>
-      <Name>{item.name}</Name>
-      <Slug>{item.symbol}</Slug>
-    </NameContainer>
-    <Price>${item.metrics.market_data.price_usd.toFixed(2)}</Price>
-    <PercentageChangeLast24hrs
-      PositiveChange={
-        item.metrics.market_data.percent_change_usd_last_24_hours > 0
-      }>
-      {item.metrics.market_data.percent_change_usd_last_24_hours.toFixed(2)}%
-    </PercentageChangeLast24hrs>
-    <Arrow>
-      <ArrowPercentage
-        source={
-          item.metrics.market_data.percent_change_usd_last_24_hours < 0
-            ? ArrowDown
-            : ArrowUp
-        }
-      />
-    </Arrow>
-  </CryptoContainer>
-);
+const Crypto: FC = ({item}: {item: CryptoTypes}) => {
```

Comment:
don't use FC, it will be deprecated you can use JSX.Element in the end of the function: 

({item}: {item: CryptoTypes}): JSX.Element